expertsystem/reaction/solving.py

```diff
@@ -146,6 +146,41 @@ def extend(
                 else:
                     self.__violated_rules[key].update(rules2)
 
+    def get_initial_state(self) -> List[Particle]:
+        if len(self.solutions) == 0:
+            raise ValueError(
+                f"No solutions in {self.__class__.__name__} object"
+            )
```

Comment:
Thats already done in `__get_first_graph()`

Reply:
Ah thanks, weird mistake